native/dlatrd.go

```diff
@@ -53,11 +53,11 @@ import (
 // H has the form
 //  I - tau * v * v^T
 // If uplo == blas.Upper,
-//  Q = H[n] * H[n-1] * ... * H[n-nb+1]
+//  Q = H(n) * H(n-1) * ... * H(n-nb+1)
```

Comment:
This seems to be still 1-indexed but (at least at the moment) my head refuses to convert it to 0-based confidently, especially together with the description of v.